src/model/helpers.rs

```diff
@@ -520,25 +520,46 @@ pub async fn get_subscriber_by_topic(
     result
 }
 
-// FIXME scaling: response not paginated
-// TODO this doesn't need to return a full subscriber
+pub struct NotifySubscriberInfo {
```

Comment:
Maybe there is a way to use only one struct?

Reply:
Would be nice if I could implement a custom `try_from` on the `scope` field, but this doesn't seem possible.

I suppose we could make the struct generic for the `scope` field, but this problem is a pattern elsewhere too and there are sometimes multiple different types that need custom `try_from`. A generic would probably be more confusing anyway than a little non-DRY code.